### ISSUE TITLE
result on validation

### ISSUE THREAD

**Issue by BuaaAlban** [mdiodato/cu-iccv-2019-learning-to-drive-winners#1]
Hi, I have reproduce your result by your code, thanks for your efforts.
But we have no lables for the test dataset , could you please provide some results on validation set to hlep me confirm whether the model I have trained is good or not?

The loss for my model is epoch5 | 858.24963 | 6.695664 | **864.945294**  on validation  with **106219** data.

The result in your paper is about **889** for model3 epoch2  on test dataset with  **279863** data, so maybe my trained model is not that good??

Another question is that, I noticed your LSTM model takes two frames' features as input, but the input sequence is in an order of [t, t-4]  instead of [t-4, t] where t is the frame index, could you give some explanation for this ?

Thanks very much if you can help!

**Comment by BuaaAlban** [comment]
Besides, you have used the resized data （160*90）in model3, but the table in your paper shows that there is only 40000 data. Have you used the whole training dataset?  Could you pleaze provide more details of the training data?
![image](https://user-images.githubusercontent.com/25258181/132273705-000fe128-b6d9-4700-b816-b32679b9ffef.png)
